src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx

```diff
@@ -72,6 +71,9 @@ export const AuctionArtworksRailFragmentContainer = createFragmentContainer(
         href
         name
         formattedStartDateTime
+        isLiveOpen
+        status
+        isPreliminary
```

Comment:
Are these fields being used anywhere in the component?

Reply:
Removed.